core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/CsvTests.kt

```diff
@@ -70,7 +70,7 @@ class CsvTests {
         df.nrow shouldBe 5
         df.columnNames()[5] shouldBe "duplicate1"
         df.columnNames()[6] shouldBe "duplicate11"
-        df["duplicate1"].type() shouldBe typeOf<String?>()
+        df["duplicate1"].type() shouldBe typeOf<Char?>()
```

Comment:
So it converts columns with single characters to Char?

Reply:
Deephaven has the same functionality, so I thought it would be a good idea to make our parsers behave the same